### PR TITLE
add xref to samples

### PR DESCRIPTION
**Pull request by rosemarymarano**
To the main README file, I added a xref to the samples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,6 +114,12 @@ however, if the metrics request is made via a load balancer or Kubernetes NodePo
 original request was made might not be the same as the instance's HTTP port. In such a case, the configuration should
 include the `restPort` configuration to tell the exporter which port to use.
 
+
+## Samples
+
+See the [samples](https://github.com/oracle/weblogic-monitoring-exporter/tree/master/samples/kubernetes) for example deployments to start Prometheus and Kibana in a Kubernetes cluster and an end-to-end sample that shows you the steps to set up monitoring for WebLogic domains using Prometheus and Grafana.
```
Comment (lilyhe123) [inline README.md]: Isn't relative link more flexible? `samples/kubernetes`

**Comment by lilyhe123** [comment]
Is this better?

## Samples
- [deployment-based sample](samples/kubernetes/deployments): This sample helps you set up and run Prometheus and Grafana with this WebLogic Monitoring Exporter. Deployment yaml files are provided to install Prometheus and Grafana.
- [chart-based sample](samples/kubernetes/end2end): This is an end-to-end sample that shows you the steps to set up monitoring for WebLogic domains using Prometheus and Grafana. Prometheus and Grafana are installed with helm chart.

File: README.md (hunk 1 above)
Comment (rosemarymarano) [reply README.md]: Yes.

**Comment by rosemarymarano** [comment]
Sure. I'll take your suggestion.

**Comment by lilyhe123** [comment]
LGTM

**Pull request merged**